nncf/experimental/onnx/graph/nncf_graph_builder.py

```diff
@@ -195,7 +195,12 @@ def create_nncf_graph(onnx_model: ModelProto) -> NNCFGraph:
                 tensor_shape = GraphConverter._get_tensor_shape(onnx_graph, output_edge)
 
                 output_node_id = nncf_graph.get_node_by_name(output_node.name).node_id
-                onnx_dtype = onnx_graph.get_edge_dtype_name(output_edge)
+                try:
+                    onnx_dtype = onnx_graph.get_edge_dtype_name(output_edge)
+                except RuntimeError:
+                    # If this edge was not added during inference of ONNX model,
+                    # we do not add it to NNCFGraph
```

Comment:
Add explanation on how this manifests itself for BatchNorm layers exported in training mode.